### PR TITLE
Fix duplicates

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [*] Update

**What category or categories does it affect?**

_Multiple_

**What?**

_Fix duplicates by changing IDs._

**Why?**

_Eliminate any potential errors due to the existence of duplicates in OEFDB._

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -695,68 +695,68 @@ Energy,Electricity,electricity-energy_source_grid_mix_residual_mix_market_based,
...
+Energy,Electricity,electricity-energy_source_grid_mix_losses,Transmission & distribution – WTT,kWh,,0.021287,,,,,,,,well_to_tank,BEIS,2021,AF,2021/12/21,Emission intensity of WTT transmission & distribution Africa (average). Well-to-Tank (WTT) conversion factors for UK and overseas electricity should be used to report the Scope 3 emissions of extraction and refining and transportation of primary fuels before their use in the generation of electricity. It should be noted that electricity generation and transmission and distribution losses have separate WTT emissions assigned to them. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (AnastasiaLobanova) [inline OpenEmissionFactorsDB.csv]: I think it is a great solution here to add it to the id! I wonder if it make sense now to have `electricity-energy_source_grid_mix_losses`? Or maybe we can just go around with `electricity-energy_source_grid_mix_losses`? It won't break our id-subid structure

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
Great job Maryam! Thank you for tackling these - I see the checker still complains that there are duplicates in these lines [780, 781] - I guess the changes added to the name column and not to the id. 

And there are also 2 ids with spaces in them - could you also correct them? I think it is ok to have one PR for them all?

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -777,8 +777,8 @@ Energy,Fuel,fuel_type_bamboo-fuel_use_stationary_combustion,Bamboo,MMBTU,93.8645
...
+Energy,Fuel,fuel_type_biodiesel_100percent-fuel_use_biodiesel_medium_and_heavy_duty_vehicles,Biodiesel (100%) - biodiesel medium and heavy-duty vehicles_8.8_fuel_efficiency,gal (US),0.012892,0.014212,9.45,0.000000044,0.000000044,not-supplied,not-supplied,12,1,fuel_combustion,GHG Protocol,2021,US,2022/01/15,Emission intensity of mobile combustion of fuel. Includes fuel consumption by vehicles that are owned or leased by the company. The primary source is EPA Emission Factors for Greenhouse Gas Inventories - March 9 2018 - WRI - GHG Protocol - Emission Factors from Cross-Sector Tools - April 2014. The CO2 factor represents the biogenic CO2 factor.,https://ghgprotocol.org/ghg-emissions-calculation-tool
```
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:780]: corrected:)

**Review by AnastasiaLobanova — APPROVED** [review]
Great!! We have green checks! Thank you Maryam!

**Comment by motin** [comment]
I merged the main branch into this one to double-check that green checks are still around :)

**Pull request merged**